conda/recipes/cudf/meta.yaml

```diff
@@ -57,7 +62,13 @@ requirements:
     - pyarrow =11
     - libcudf ={{ version }}
     - rmm ={{ minor_version }}
+    {% if cuda_major == "11" %}
     - cudatoolkit ={{ cuda_version }}
+    {% else %}
+    - cuda-nvrtc
+    - libcufile  # [linux64]
```

Comment:
Not libcufile-dev? Based on our discussion from yesterday I guess this one is even less necessary though since we just want to make libcufile a run requirement right? No build req whatsoever?